docs/site-replication/run-ssec-object-replication.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC2120
+exit_1() {
+	cleanup
+
+	echo "minio1 ============"
+	cat /tmp/minio1_1.log
+	echo "minio2 ============"
+	cat /tmp/minio2_1.log
+
+	exit 1
+}
+
+cleanup() {
+	echo -n "Cleaning up instances of MinIO ..."
+	pkill minio || sudo pkill minio
+	pkill -9 minio || sudo pkill -9 minio
+	rm -rf /tmp/minio{1,2}
+	echo "done"
+}
+
+cleanup
+
+export MINIO_CI_CD=1
+export MINIO_BROWSER=off
+export MINIO_ROOT_USER="minio"
+export MINIO_ROOT_PASSWORD="minio123"
+
+# Create certificates for TLS enabled MinIO
+echo -n "Setup certs for MinIO instances ..."
+wget -O certgen https://github.com/minio/certgen/releases/latest/download/certgen-linux-amd64 && chmod +x certgen
+./certgen --host localhost
+mkdir -p ~/.minio/certs
+mv public.crt ~/.minio/certs || sudo mv public.crt ~/.minio/certs
+mv private.key ~/.minio/certs || sudo mv private.key ~/.minio/certs
+echo "done"
+
+# Start MinIO instances
+echo -n "Starting MinIO instances ..."
+minio server --address ":9001" --console-address ":10000" /tmp/minio1/{1...4} >/tmp/minio1_1.log 2>&1 &
+minio server --address ":9002" --console-address ":11000" /tmp/minio2/{1...4} >/tmp/minio2_1.log 2>&1 &
+echo "done"
+
+if [ ! -f ./mc ]; then
+	echo -n "Downloading MinIO client ..."
+	wget -O mc https://dl.min.io/client/mc/release/linux-amd64/mc &&
+		chmod +x mc
+	echo "done"
+fi
+
+sleep 10
+
+export MC_HOST_minio1=https://minio:minio123@localhost:9001
+export MC_HOST_minio2=https://minio:minio123@localhost:9002
+
+# Prepare data for tests
+echo -n "Preparing test data ..."
+mkdir -p /tmp/data
+echo "Hello world" >/tmp/data/plainfile
+echo "Hello from encrypted world" >/tmp/data/encrypted
+for index in {1..10000000}; do echo "$index - The quick brown fox jumps over the lazy dog" >>/tmp/data/defpartsize; done
```

Comment:
this is too slow @shtripat please use something like `shred` or `cat /dev/urandom`

Reply:
ok

Reply:
reason was to have some readable data so that at target end we get the data and verify if data got decrypted well